docs/understanding-airbyte/operations.md

```diff
@@ -0,0 +1,51 @@
+# Sync Operations
+
+Airbyte [connections](connections/README.md) support configuring additional transformations that execute after the sync. Useful applications could be:
+
+- Customized normalization to better fit the requirements of your own business context.
+- Business transformations from a technical data representation into a more logical and business oriented data structure. This can facilitate usage by end-users, non-technical, executives for Business Intelligence dashboard and report.
```

Comment:
```suggestion
- Business transformations from a technical data representation into a more logical and business oriented data structure. This can facilitate usage by end-users, non-technical operators, and executives looking to generate Business Intelligence dashboards and reports.
```